api-guide.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Webhooks API guide
+sidebar_label: API guide
+sidebar_position: 30
+description: Find technical details about integrating with the Webhooks API.
+pagination_prev: Null
+pagination_next: Null
+---
+
+# API guide
+
+Webhooks alert you to your API request updates (e.g., payment *Authorized*, *Cancelled*, *Expired*, and
+[others](https://developer.vippsmobilepay.com/docs/APIs/webhooks-api/events/)).
+
+You can register to receive webhooks for specific events (e.g., *Cancelled* payment requests)
```

Comment:
This paragraph and the one above seems like it should be combined, they kinda say the same thing.

Reply:
I deleted the 2nd one.